### PR TITLE
Modify configuration files for adminrouter

### PR DESCRIPTION
**Pull request by KostiantynOleksiienko**
## High-level description

This pr modifies build script and config files for dcos-adminrouter.

## Corresponding DC/OS tickets (required)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS_OSS-5734) JIRA title / short description.

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by KostiantynOleksiienko** [comment]
@mesosphere-mergebot changelog-not-required

**Comment by KostiantynOleksiienko** [comment]
@mesosphere-mergebot request-review @DominikDary

**Comment by KostiantynOleksiienko** [comment]
@mesosphere-mergebot request-review @jongiddy

File: packages/adminrouter/windows.build.ps1 [hunk 1]
```diff
@@ -20,8 +19,10 @@ function Unzip {
 
 Unzip "c:\pkg\src\OpenResty\openresty-1.15.8.2-win64.zip" "c:/Temp/openresty"
 
-ls "C:\pkg\build\extra\src\"
```
Comment (jongiddy) [inline packages/adminrouter/windows.build.ps1]: Error caused by difference of `C:\Temp` and `$TempFolder`?

Comment (KostiantynOleksiienko) [reply packages/adminrouter/windows.build.ps1]: Thank you, fixed it.

**Pull request merged**